user/caribou/module/src/DSO9254AEventConverter.cc

```diff
@@ -0,0 +1,448 @@
+#include "CaribouEvent2StdEventConverter.hh"
+#include "utils/log.hpp"
+#include "TFile.h"
+#include "TH1D.h"
+#include "TGraph.h"
+#include "time.h"
+
+using namespace eudaq;
+
+namespace{
+  auto dummy0 = eudaq::Factory<eudaq::StdEventConverter>::
+    Register<DSO9254AEvent2StdEventConverter>(DSO9254AEvent2StdEventConverter::m_id_factory);
+}
+
+
+bool    DSO9254AEvent2StdEventConverter::m_configured(0);
```

Comment:
no format provided in eudaq :shrug:

Reply:
Only because it's not enforced on an old code base that didn't have checks in place from the beginning doesn't mean you should continue to destroy formatting. :wink:

Reply:
I agree - but I will not format this file before we completed the review on functionality